src/schema/system/index.js

```diff
@@ -0,0 +1,21 @@
+import { GraphQLObjectType } from "graphql"
+
+import SystemTime from "./time"
+
+const SystemType = new GraphQLObjectType({
+  name: "System",
+  fields: {
+    time: SystemTime,
+  },
+})
+
+const System = {
```

Comment:
We could nest other fields under this, such as `status` (which is currently at a separate root note)